src/main/java/com/group12/LaunchInterceptorConditionCollection.java

```diff
@@ -100,6 +100,33 @@ public boolean LIC3(List<Point> points, double area1) throws IllegalArgumentExce
         return false;
     }
 
+    /**
+     * LIC #5 check if there exists a set of two consecutive data points where the x-value of the latter data point is
+     * greater than the former data point
+     * @param points list of radar echos ({@link Point})
+     * @return true if there exists a set of two consecutive data points where the x-value of the latter data point is
+     * greater than the former one.
```

Comment:
The code is correct, but the description here seems to be in the reverse order.
Is it: "the x-value of the former data point is greater than the latter one"?